src/wsdiscoveryclient.cpp

```diff
@@ -46,7 +46,9 @@ WSDiscoveryClient::~WSDiscoveryClient() = default;
 void WSDiscoveryClient::start()
 {
     bool rc = m_soapUdpClient->bind(DISCOVERY_PORT, QAbstractSocket::ShareAddress);
-    Q_ASSERT(rc);
+    if (!rc) {
+        qWarning() << "start()->KDSoapUdpClient::bind() returned false when binding to port" << DISCOVERY_PORT;
+    }
```

Comment:
I don't think this is a good idea. bind() is not supposed to fail. If it does then I want a hard error during debugging and tests. 

In KDSoap there is a todo in KDSoapUdpClient.cpp for joining the IPv4 and IPv6 socket into a single dual-stack socket. Maybe this would solve your problem? (It is on my todo list, but you could beat me to it ;-)

Reply:
I agree.  I might be able to find some time in the near future to make the changes to KDSoapUdpClient.cpp.  In the short the warnings allowed me to debug without an assert.